tasks/72_carrying/carrying.js

```diff
@@ -0,0 +1,24 @@
+export function addTogether(a, b) {
+  if (a && b) {
+    if (typeof a === 'number' && typeof b == 'number') {
+      return a + b;
+    } else {
+      return undefined;
+    }
+  }
+
+  if (a || b) {
```

Comment:
Why do you check first argument? It is possible to call `addTogether(undefined, 1)`?